src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -9,7 +9,6 @@ import {
 } from '../../../../types/component/ChatList/ChannelImage.types';
 import GroupIcon from '../../../assets/group-icon.png';
 import CommunityIcon from '../../../assets/hashtag.png';
-import {GROUP_INFO} from '../../../hooks/core/constant';
 import {channelImageStyles as styles} from './ChannelImage.style';
 
 const ChannelImage = ({children}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `GROUP_INFO` has been removed. If `GROUP_INFO` is used elsewhere in the code, this could cause a reference error.
```diff
- import {GROUP_INFO} from '../../../hooks/core/constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->